src/test/resources/jsonSchema/uktrSchema/Response/UKTR_201.json

```diff
@@ -1,41 +1,19 @@
 {
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "$ref": "#/definitions/Welcome3",
+  "$ref": "#/definitions/Welcome7",
   "definitions": {
-    "Welcome3": {
+    "Welcome7": {
       "type": "object",
       "additionalProperties": false,
       "properties": {
         "success": {
-          "$ref": "#/definitions/Success"
+          "type": "boolean"
         }
       },
       "required": [
         "success"
       ],
-      "title": "Welcome3"
-    },
-    "Success": {
-      "type": "object",
-      "additionalProperties": false,
-      "properties": {
-        "processingDate": {
-          "type": "string",
-          "format": "date-time"
-        },
-        "formBundleNumber": {
-          "type": "string"
-        },
-        "chargeReference": {
-          "type": "string"
-        }
-      },
-      "required": [
-        "chargeReference",
-        "formBundleNumber",
-        "processingDate"
-      ],
-      "title": "Success"
+      "title": "Welcome7"
```

Comment:
why is this called `Welcome7`? Also, I don't think this is the correct response payload

Reply:
or maybe it isn't correct based on the new branch that I have. I think you'll have to change this quite soon